proto/streamlit/proto/Block.proto

```diff
@@ -29,6 +29,7 @@ message Block {
     TabContainer tab_container = 6;
     Tab tab = 7;
     ChatMessage chat_message = 9;
+    Popover popover = 10;
```

Comment:
`// Next ID: 10` [in the bottom of the file](https://github.com/streamlit/streamlit/pull/7908/files#diff-8ad066877d960b2df08c08e99a9580848d327a45b9d2eec4bc913a89e7de537fR89) should be incremented to `11` -> `// Next ID: 11`

Reply:
Fixed